vigor/helpers.py

```diff
@@ -0,0 +1,5 @@
+from django.http import HttpResponse
+
+def is_authenticated(request):
```

Comment:
@juliancoleman I think we should change this to `redirect_if_authenticated`, it's not very clear what the action of `is_authenticated` does.